client/src/js/components/header.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { updateQueryInput } from '../actions/index';
+import { calculateOverAllRates } from '../../../../helpers/clientHelplers';
+
+const mapStateToProps = state => ({
+  roomTotalReviewNumber: state.roomTotalReviewNumber,
+  overAllRating: calculateOverAllRates(state.overAllRating),
```

Comment:
overall is one word. should rename to calculateOverallRates

Reply:
I will incorporate the edit in next pr.